core/category_api.php

```diff
@@ -85,6 +85,10 @@ function category_ensure_exists( $p_category_id ) {
  * @return bool True if the category exists, false otherwise.
  */
 function category_exists_in_project( $p_category_id, $p_project_id ) {
+	$t_allow_no_category = config_get( 'allow_no_category', null, null, $p_project_id );
+	if( $t_allow_no_category && $p_category_id == 0 ) {
+		return true;
+	}
```

Comment:
To avoid some `config_get` calls you could change to
```
	if( $p_category_id == 0 && config_get( 'allow_no_category', null, null, $p_project_id ) ) {
		return true;
	}	
```

Reply:
Thanks for the hint.